MachObfuscatorTests/OptionsTests/Options_Tests.swift

```diff
@@ -54,7 +64,7 @@ class Options_Tests: XCTestCase {
                           argv: argv)
 
         // Then
-        XCTAssertFalse(sut.quiet)
+        XCTAssertTrue(sut.quiet)
```

Comment:
I did something stupid here 🤦‍♂️ Thanks for fixing that!

Reply:
`optind = 1` does the trick